pkg/hintrunner/hinter/scope.go

```diff
@@ -3,37 +3,58 @@ package hinter
 import (
 	"fmt"
 	"math/big"
+
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
 // ScopeManager handles all operations regarding scopes:
 // - Creating a new scope
 // - Exiting current scope
 // - Variable declaration and assignment inside a certain scope
 // - Accessing variable values
+
+type ScopeValue struct {
+	Type        int
+	FeltValue   *fp.Element
```

Comment:
Do we need Value suffix in the type name?

Reply:
Already changed it.